doc/classes/EditorProperty.xml

```diff
@@ -202,6 +202,11 @@
 				Emit it if you want to key a property with a single value.
 			</description>
 		</signal>
+		<signal name="property_overridden">
+			<description>
+				Emitted when a setting is overriden for the current project.
```

Comment:
```suggestion
				Emitted when an editor setting is overridden for the current project.
```
or, well, achtually:
```suggestion
				Emitted when a setting override for the current project is requested.
```
but the surrounding descriptions are comparatively lacking in this detail.
Although it's a bit nasty to expose the signal, mostly because `can_override` is not exposed as a property, but unfortunately signals cannot be hidden from the API. Eh. It's not too different from `property_pinned` and `property_favorited`.

Reply:
*Technically* signals can be "hidden" with `add_user_signal()`,  but it requires adding the signal for each instance individually.

Reply:
Yep, we have only 3 uses right now.
Hiding underscored signals makes more sense, idk why it's not done yet.